t/t6026-merge-attr.sh

```diff
@@ -180,4 +180,28 @@ test_expect_success 'up-to-date merge without common ancestor' '
 	)
 '
 
+cat >./custom-merge <<\EOF
+#$SHELL_PATH
+sleep 1 &
+EOF
+chmod +x ./custom-merge
```

Comment:
Let's use the `write_script` function instead. See e.g. https://github.com/git/git/blob/v2.8.3/t/t0001-init.sh#L91-L98

As to the name, let's call it `sleep-one-sec.sh`?